api/services/taxonomyService.ts

```diff
@@ -0,0 +1,27 @@
+import { makeSCOSRequest } from './scosService';
+import getLogger from '../utils/logger';
+import TaxonomyDto from '../models/dto/taxonomyDto';
+import ScosSubcategoryDto from '../models/scosApi/scosSubcategoryDto';
+import ScosCategoryDto from '../models/scosApi/scosCategoryDto';
+
+const { CATEGORIES_TABLE } = process.env;
+
+const log = getLogger('taxonomyService');
+
+async function getSubcategories(id: string): Promise<TaxonomyDto[]> {
+  log.debug(`Getting subcategories of category id ${id}`);
+
+  const query = `SELECT subcat FROM ${CATEGORIES_TABLE} WHERE categoryid = '${id}'`;
```

Comment:
Why don't we need to break this out into a repository file? Is it because we only expect to do this one query for this service so it would be overkill?

Reply:
Technically, this isn't querying a database (mostly). This query is sent to the SCOS API... it _could_ be pulled out into a separate